pyscf/__init__.py

```diff
@@ -98,6 +98,7 @@ def _get_void_ptr(arr):
 from pyscf import gto
 from pyscf import scf
 from pyscf import ao2mo
+from pyscf import hessian
```

Comment:
why do you need to import hessian in the global `__init__.py` file?

Reply:
`.Hessian()` is not defined in SCF. It was injected when import hessian. When we call mf.Hessian(), we have to `from pyscf import hessian` first for now.

Reply:
hessian module will lead an import chain that loads many irrelevant modules. Please remove this line in this PR. It can be discussed in another issue/PR whether these modules should be imported in the global `__init__.py`

Reply:
OK. Let me create an issue for this.